dts/riscv/espressif/esp32c6/esp32c6_common.dtsi

```diff
@@ -75,9 +75,14 @@
 		};
 
 		sramlp: memory@50000000 {
+			#address-cells = <1>;
+			#size-cells = <1>;
 			compatible = "zephyr,memory-region", "mmio-sram";
 			reg = <0x50000000 DT_SIZE_K(16)>;
 			zephyr,memory-region = "SRAMLP ";
+			shmlp: memory@0 {
```

Comment:
inconsistent spacing

Reply:
why?

Reply:
No newline before the child node here but there is in the rest of the file